Makefile

```diff
@@ -20,6 +20,9 @@ $(VENV)/bin/pytest: $(VENV)/bin/python requirements.txt
 docs:
 	tox -e docs
 
+live-docs: $(VENV)/bin/sphinx-build
+	SPHINXBUILD=../$(VENV)/bin/sphinx-build sphinx-autobuild docs docs/_build/html
+
```

Comment:
What I've meant that this should become something like:
```
live-docs:
	tox -e live-docs
￼```
But just leave it out there for now.
I might pick it up the next time working on docs (and hopefully remembering it).

Reply:
sphinx-autobuild starts a long-running web server process, that serves rendered documentation and automatically reloads it on changes. 

I see no advantage running it under tox, as it would probably block "clean" tox run and cause a timeout.